### DISCUSSION TITLE
Error in BMI value in MoBi

### DISCUSSION THREAD

**Discussion by mags0728** [Open-Systems-Pharmacology/Forum#1789] [Questions & Problems]
Hello there, 

I imported a pkml file into MoBi from PK-Sim where I'm trying to implement changes in protein binding. When I make the changes in MoBi and try to run the simulations, it gives this error "Value for BMI should be greater than or equal to 4 kg/m2". The population I'm working with is an infant population and in PK-Sim, the Individual in this simulation has the following parameters: age=1 yo; weight=10kg; height=76 cm; and bmi=17.31 kg/m2. The simulation runs in PK-Sim and when I import in MoBi before I implement the changes in protein binding. 

I have previously successfully implemented changes I wanted in MoBi before with older individuals (greater than 2 years old) so I'm wondering what am I doing wrong here. 

Any thoughts on what is happening or how to troubleshoot? Thank you!
![MoBi_BMI_error](https://github.com/Open-Systems-Pharmacology/Forum/assets/97306838/ed2dbec9-928c-4a67-a256-70b59b10a131)

**Comment by Yuri05** [comment]
Can you upload your MoBi project here?

**Reply by mags0728** [reply]
Hi Yuri05, I did more troubleshooting and found that I would get this error if I did "Allow Aging" in the PK-Sim simulation prior to importing it to MoBi. Once I unchecked "Allow Aging", the issue resolved. 

All is good now. Thanks!

**Reply by Yuri05** [reply]
Nice to hear.
However, aging should not prevent the simulation from running...
Would be still good if you could upload your project for investigation.
